### PR TITLE
AO3-5889 Paginate pseuds using default per page setting

### PR DESCRIPTION
**Pull request by neuroalien**
# Pull Request Checklist

* [x] Have you read ["How to write the perfect pull request"](https://github.blog/2015-01-21-how-to-write-the-perfect-pull-request/)?
* [x] Have you read the [contributing guidelines](https://github.com/otwcode/otwarchive/blob/master/CONTRIBUTING.md)?
* [ ] Have you added [tests for any changed functionality](https://github.com/otwcode/otwarchive/wiki/Automated-Testing)?
* [x] Have you added the [Jira](https://otwarchive.atlassian.net) issue number
  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
* [x] Do you have fewer than 5 pull requests already open? If not, please wait
  until they are reviewed and merged before creating new pull requests.

If cucumber tests are required, I would appreciate being paired with a coder who is okay with writing cucumber features, please.

## Issue

https://otwarchive.atlassian.net/browse/AO3-5889

## Purpose

Adds pagination to the pseuds page.

## Testing Instructions

How can the Archive's QA team verify that this is working as you intended?

If you have a Jira account with access, please update or comment on the issue
with any new or missing testing instructions instead.

## References

Are there other relevant issues/pull requests/mailing list discussions?

## Credit

What name and pronouns should we use to credit you in the [Archive of Our Own's Release Notes](https://archiveofourown.org/admin_posts?tag=1)?

alien, they/them

### CODE REVIEW

File: app/controllers/pseuds_controller.rb [hunk 1]
```diff
@@ -14,7 +14,7 @@ def load_user
   # GET /pseuds.xml
   def index
     if @user
-      @pseuds = @user.pseuds
+      @pseuds = @user.pseuds.paginate(page: params[:page])
```
Comment (sarken) [inline app/controllers/pseuds_controller.rb:17]: Should we set this to use the `alphabetical` scope? I noticed when I was playing with tests I'd sometimes get the wrong pseud on the second page if I didn't add that.

Speaking of tests, I added this right after https://github.com/otwcode/otwarchive/blob/ba94c04580de2bac4c7a932e33a11fad4de7e06d/features/other_a/pseuds.feature#L162
```
  When I go to my pseuds page
  Then I should not see "Zaphod (Zaphod)" within "ul.pseud.index"
    But I should see "Agrajag (Zaphod)" within "ul.pseud.index"
    And I should see "Betelgeuse (Zaphod)" within "ul.pseud.index"
    And I should see "Slartibartfast (Zaphod)" within "ul.pseud.index"
    And I should see "Next" within ".pagination"
  When I follow "Next" within ".pagination"
  Then I should see "Zaphod (Zaphod)" within "ul.pseud.index"
```

File: app/views/pseuds/index.html.erb [hunk 2]
```diff
@@ -11,6 +11,10 @@
 <% end %>
 <!--/subnav-->
 
+<!--subnav-->
+<%= will_paginate @pseuds %>
```
Comment (sarken) [inline app/views/pseuds/index.html.erb:15]: Wait, sorry, one more comment -- can you move `<%= will_paginate @pseuds %>` up above the `<!--/subnav-->` on line 12? We usually just have one subnav comment on the top and one on the bottom.

File: app/controllers/pseuds_controller.rb (hunk 1 above)
Comment (neuroalien) [reply app/controllers/pseuds_controller.rb:17]: Sure! Do we want `alphabetical` or `default_alphabetical`? Presumably what we use now will stick for the foreseeable 🙈

Comment (neuroalien) [reply app/controllers/pseuds_controller.rb:17]: Thanks for the spec! 🙌

Comment (sarken) [reply app/controllers/pseuds_controller.rb:17]: alphabetical is good! That's what it seems to be on staging

**Review by sarken — APPROVED** [review]
🎉

**Pull request merged**